bouncer/tests/polkadot_runtime_update.ts

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env -S pnpm tsx
+import { testPolkadotRuntimeUpdate } from '../shared/polkadot_runtime_update';
+import { getChainflipApi, observeEvent, runWithTimeout } from '../shared/utils';
+
+let broadcastAbortedCount = 0;
+
+async function observeBroadcastAborted(): Promise<void> {
+  for (;;) {
+    await observeEvent(':BroadcastAborted', await getChainflipApi());
+    console.log('Broadcast aborted');
+    broadcastAbortedCount++;
+  }
+}
+
+async function main(): Promise<void> {
+  observeBroadcastAborted();
```

Comment:
I think we should write this test such that if we get an aborted, we throw an error and fail the test. When I tried it seemed to just log "Broadcasts aborted"